SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -292,6 +292,8 @@ const FString SCHEMA_DATABASE_ASSET_PATH = TEXT("/Game/Spatial/SchemaDatabase");
 
 const FString DEV_LOGIN_TAG = TEXT("dev_login");
 
+const int64 INVALID_RPC_ID = -1;
```

Comment:
```suggestion
const uint64 INVALID_RPC_ID = -1;
```

Reply:
I think we should defo make it uint64, that was an error on my part. 
I think we should use -1 as I believe this will set INVALID_RPC_ID to the highest possible value of an uint64 which is probably safer than 0.

Reply:
0 is an invalid rpc ID though. uint64_max is valid.

Reply:
True, this a minor detail and I do not care either way but... I think it is more likely that 0 becomes a valid rpc ID at some point than we ever reach uint64_max rpcs. If we do reach uint64_max we have a bigger problem no?

Reply:
I think it's impossible the 0 can ever be a valid RPC ID unless we make the ID an option field, which seems unlikely.

But trying to make this -1 means if you actually compare invalid ID to the actually invalid ID that can be stored in schema it will appear different.